packages/beacon-node/src/chain/validatorMonitor.ts

```diff
@@ -1061,3 +1068,221 @@ export class RootHexCache {
     return root;
   }
 }
+
+function createValidatorMonitorMetrics(register: RegistryMetricCreator) {
+  return {
+    validatorsConnected: register.gauge({
+      name: "validator_monitor_validators",
```

Comment:
maybe something we wanna do for our 2.0 release is rename these metrics

not sure why they are not prefixed with `lodestar_` but this should be

```suggestion
      name: "lodestar_validator_monitor_validators",
```

Reply:
(can ignore this for now, but could create an issue to keep track of the idea)